### PR TITLE
Bug fixes to Image block

### PR DESCRIPTION
**Pull request by hissy**
* Can't save the `forceImageToMatchDimensions` value
* Can't get the size of the hover image

### CODE REVIEW

**Comment by Remo** [comment]
I still can't set "Scale Image" to "Automatically", doesn't get saved?
I also can't set the size of the thumbnail, it will be saved but the output isn't resized.

**Comment by hissy** [comment]
Well, the problems were fixed by #1904 ?

**Comment by Remo** [comment]
yes, but the $fos fix of yours was still needed, wasn't it?

File: web/concrete/core/controllers/blocks/image.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ function getContentAndGenerate($align = false, $style = false, $id = null) {
 			$img .= ($style) ? "style=\"{$style}\" " : '';
 			if($this->fOnstateID != 0) {
 				$fos = $this->getFileOnstateObject();
-				$fullPathOnstate = $f->getPath();
+				$fullPathOnstate = $fos->getPath();
```
Comment (hissy) [inline web/concrete/core/controllers/blocks/image.php]: Yes, this fix is still valid and needed

**Pull request merged**